bucket/ironwail.json

```
@@ -0,0 +1,87 @@
+{
+    "version": "0.6.0",
+    "description": "Fork of QuakeSpasm with a focus on high performance instead of maximum compatibility, with a few extra features sprinkled on top",
+    "homepage": "https://github.com/andrei-drexler/ironwail",
+    "license": "GPL-2.0-only",
```

I checked license headers, and it's "or later":

```suggestion
    "license": "GPL-2.0-or-later",
```